File: corpus/typescript/fixture_11.ts
type CartItem11 = { price: number; qty?: number };
export function calculateDiscount11(cartItems: CartItem11[], taxRate: number): number {
  const totalAmount = cartItems.reduce((sum, item) => sum + item.price * (item.qty ?? 1), 0);
  return totalAmount > 110 ? totalAmount * (1 + taxRate) : totalAmount;
}

export class PriceBook11 {
  lookupPrice(sku: string): number { return sku.length + 11; }
}

File: corpus/go/fixture_11.go
package fixture11

type PriceBook11 struct {
    Multiplier float64
}

func CalculateDiscount11(cartItems []float64, taxRate float64) float64 {
    totalAmount := 0.0
    for _, price := range cartItems {
        totalAmount += price
    }
    if totalAmount > 110.0 {
        return totalAmount * (1.0 + taxRate)
    }
    return totalAmount
}
